packages/components/style.css

```diff
@@ -1,23 +1,26 @@
 @layer l-base, l-nextra, l-components;
 
-@import 'tailwindcss/base' layer(l-base);
+/*@import 'tailwindcss/base' layer(l-base);*/
```

Comment:
I think we don't need this, cc @hasparus since nextra already imports preflights styles

Reply:
https://github.com/shuding/nextra/blob/4616cf08674d9c5fd4f61cc96051b783b804fc73/packages/nextra-theme-docs/src/style.css#L1

Reply:
Oh, so the preflight from Tailwind lands in `l-nextra` layer. That's fine. We'll drop some duplicate CSS. (Actually... I wonder if cssnano deduplicated it previously).

Reply:
When I removed it from `nextra.site` https://github.com/shuding/nextra/pull/3641 I got -6 B, so I guess it deduplicated already well